docs/CHANGELOG.md

```diff
@@ -36,7 +26,28 @@ Test fixtures for use by clients are available for each release on the [Github r
 
 ### 📋 Misc
 
-- 🐞 WELDed the EEST tox environments relevant to producing documentation into EELS, and added a tool to cleanly add codespell whitelist entries. ([#1695](https://github.com/ethereum/execution-specs/pull/1659)).
+### 🧪 Test Cases
+
+## [v5.4.0](https://github.com/ethereum/execution-spec-tests/releases/tag/v5.4.0) - 2025-12-05
```

Comment:
I assume we shouldn't merge this until we have the actual release, in case the date here changes and so we align on the tag.